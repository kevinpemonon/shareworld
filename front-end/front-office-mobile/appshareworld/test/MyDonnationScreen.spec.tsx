import React from 'react';
import MyDonnationScreen from '../src/screens/profile/MyDonnationsScreen';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<MyDonnationScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  