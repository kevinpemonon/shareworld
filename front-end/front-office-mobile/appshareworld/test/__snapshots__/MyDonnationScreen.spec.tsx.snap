// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "marginBottom": 20,
      "padding": 10,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#e9383f"
      hidesWhenStopped={true}
      size="large"
    />
  </View>
</View>
`;
